### DISCUSSION TITLE
How are you supposed to log into Git using the terminal? STUCK FOR HOURS!

### DISCUSSION THREAD

**Discussion by juancarlosortizr** [community/community#52270] [New to GitHub]
I've been stuck for hours simply trying to log into Git using the terminal, namely using the `gh auth login` command.

It takes me to a browser and says to write a code "displayed on my device". Nothing is displayed on my device. No SMS. Nothing on my phone. No email. Forums say it's already on my clipboard. No it's not. No 2FA either on my authenticator app.

I tried using the SSH option instead of the HTTPS one but it's even more confusing.

Please help!

**Comment by anshtiwatne** [comment]
The code "displayed on device" is actually displayed in your terminal when you run `gh auth login`

Here's a screenshot from my terminal where I have underlined the code for example:
<img alt="image" src="https://user-images.githubusercontent.com/83647366/232228513-419b812d-d888-4766-9786-966c859c8622.png">
Note that in this screenshot I have use HTTPS as my preferred protocol

Try pasting the code from your terminal into the browser page, and you should be logged in successfully
